src/element-style-observer.js

```diff
@@ -96,17 +97,40 @@ export default class ElementStyleObserver {
 	/**
 	 * Called the first time observe() is called to initialize the target.
 	 */
-	#init() {
+	#init () {
 		if (this.#initialized) {
 			return;
 		}
 
 		let firstTime = this.constructor.all.get(this.target).size === 1;
 		this.updateTransition({ firstTime });
 
+		// Initialize transition behavior for the main document
+		this.#setTransitionBehavior(this.target.ownerDocument);
```

Comment:
I don't think this is necessary: It will be added once we observe a node in the light DOM, and if we never do, then we don't need it. Additionally, it's not desirable when Style-Observer is used within components, as they don't typically want to leak stuff to the light DOM unless absolutely necessary.

Reply:
Got it. Thanks!